src/Symfony/Bundle/TwigBundle/TemplateIterator.php

```diff
@@ -29,16 +29,19 @@ class TemplateIterator implements \IteratorAggregate
     private \Traversable $templates;
     private array $paths;
     private ?string $defaultPath;
+    private array $fileNames;
 
     /**
-     * @param array       $paths       Additional Twig paths to warm
-     * @param string|null $defaultPath The directory where global templates can be stored
+     * @param array         $paths       Additional Twig paths to warm
+     * @param string|null   $defaultPath The directory where global templates can be stored
+     * @param string[]|null $fileNames   Pattern of file names
```

Comment:
Not nullable. 
Empty array = no filter.

Reply:
So, null should be removed here?